Wright_Will/Assignments/Sum_List.py

```diff
@@ -0,0 +1,2 @@
+a = [1,2,5,10,255,3]
+print sum(a)
```

Comment:
Here too 😅

In the future, you of course will use built in functions; but in the fundamentals chapter, they want you to _write_ the sum method